instrumentation-api-caching/build.gradle.kts

```diff
@@ -36,19 +36,4 @@ tasks {
 
     minimize()
   }
-
-  named("jar") {
-    enabled = false
```

Comment:
This was to avoid publishing both a normal jar and the -all one. But it's added complication without much benefit so the change seems fine to me.